llvmlite/binding/executionengine.py

```diff
@@ -53,15 +55,25 @@ def __init__(self, ptr, module):
         module._owned = True
         ffi.ObjectRef.__init__(self, ptr)
 
-    def get_pointer_to_global(self, gv):
-        # XXX getPointerToGlobal is deprecated for MCJIT,
-        # getGlobalValueAddress should be used instead.
+    def get_pointer_to_function(self, gv):
+        warnings.warn(".get_pointer_to_function() is deprecated.  Use "
+                      ".get_function_address() instead.", DeprecationWarning)
         ptr = ffi.lib.LLVMPY_GetPointerToGlobal(self, gv)
         if ptr is None:
             raise ValueError("Cannot find given global value %r" % (gv.name))
         return ptr
 
-    get_pointer_to_function = get_pointer_to_global
```

Comment:
The name ``get_pointer_to_global()``  implies it works on any global values but it does not.  It gives incorrect result for non-function global values.  With ``get_pointer_to_function()``, the name implies that it is expecting a function value and it works correctly.  I intentionally break the API to help user avoid getting a segfault.